src/arrayMethodSort.js

```diff
@@ -4,8 +4,18 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let n = i + 1; n < this.length; n++) {
+        if (compareFunction(this[i], this[n]) > 0) {
```

Comment:
The compareFunction should return a number less than 0, 0, or greater than 0. Ensure that the compareFunction adheres to this contract for all possible inputs.